daemon/lyrics.py

```diff
@@ -404,7 +404,7 @@ def _expand_patterns(self, metadata):
             try:
                 path = expand_path(path_pat, metadata)
             except osdlyrics.pattern.PatternException:
-                logger.exception('expand gg')
+                logging.exception('expand gg')
```

Comment:
Any reason why we shouldn't keep it?

Reply:
Since other PatternException(s) are not logged, either. I forgot to remove this after I debug something.

To keep the consistency, either "no logging at all" and "logging all" are better the "logging some".

If you want to keep that, please:
1. Add logging to other 3 PatternExceptions too.
2. Change the logging message.
3. As these exceptions are tolerable, maybe logging level DEBUG is sufficient. (`logging.debug(msg, exc_info=True)`)

Reply:
OK, I removed the pattern exception. New version pushed.